test/test_images/sendevents/main.go

```diff
@@ -138,6 +142,9 @@ func main() {
 		}
 		event.SetType(eventType)
 		event.SetSource(eventSource)
+		if eventExtensionName != "" {
```

Comment:
Do we allow empty string "" as the extension name? I checked the source code in the receiver.go, it seems like we can use empty string as the extension name. I'm wondering whether it's ok that I use "" to indicate there is no extension?

Reply:
you are right, it will be OK to check for eventExtensions != ""